spec/features/events_spec.rb

```diff
@@ -73,12 +73,10 @@
     
     click_link "Email"
     page.should have_content "Email the participants of Death Star's destruction celebration"
```

Comment:
Let's simplifv this test by splitting it up.

Have one test that ensures events have a email link when the user has the tsp role. And that clicking the link directs the user to the email view.
That test will complement testing access to the page ("User logged in is not a tsp member" and "When a user is not logged in").

That way we can focus in this test on testing that a user can send emails. So here we can directly go to the email view and do the stuff that happens in line 77 and following.

I would recommend doing this via ``` git rebase -i 704e604719d61c11f0f1174c4a5126aae76ba286^ ```. Then change 'pick 704e60471' to 'edit 704e60471', do your changes, do a git add, and git rebase --continue.